crates/common/flockfile/src/unix.rs

```diff
@@ -86,7 +86,7 @@ impl Flockfile {
                 })?;
 
         // Convert the PID to a string
-        let pid_string = format!("{}", Pid::this());
+        let pid_string = format!("{}", get_pid());
```

Comment:
Indeed, there is no point to use `nix::unistd::Pid`. As the process id is only used to be printed, `std::process::id()` is all what we need.

```suggestion
        let pid_string = format!("{}", std::process::id());
```

Reply:
updated